content/en/users/cloud-compute/ec3/apps/_index.md

```diff
@@ -41,16 +41,30 @@ please select Mesos + Marathon + Chronos from the list of available LRMS.
 To deploy
 [Serverless computing for data-processing applications](https://www.egi.eu/about/newsletters/serverless-computing-for-data-processing-applications-in-egi/)
 in EGI, please select OSCAR from the list of LRMS (Local Resource Management
-System). Use cases of applications that use the OSCAR framework for
-event-driven high-throughput processing of files:
+System). OSCAR supports data-driven serverless computing for file-processing applications. 
+Services will be triggered in response to a file upload to an object storage backend in order 
+to execute a user-defined shell script inside a container provisioned out of an user-defined 
+Docker image. These will be orchestrated as a Kubernetes batch jobs. The output data will be uploaded 
```

Comment:
This sentence seems a bit too convoluted/complex, maybe we could try to simplify in a way similar the the proposition below (I may have misunderstood something, so please adapt):
```suggestion
A file upload, to an object storage backend, will trigger the execution of a chosen shell script running inside a user-defined container. These will be orchestrated as a Kubernetes batch jobs. The output data will be uploaded 
```